### DISCUSSION TITLE
OSM organized editing

### DISCUSSION THREAD

**Discussion by ynux** [EqualStreetNames/equalstreetnames#759] [Q&A] [answered]
@kr4lt points out in https://github.com/EqualStreetNames/equalstreetnames-munich/issues/1 :

> Your effort may possibly be considered organised editing (https://wiki.osmfoundation.org/wiki/Organised_Editing_Guidelines)

So should this project be added to https://wiki.openstreetmap.org/wiki/Organised_Editing/Activities ?

**Comment by jbelien** [comment, marked as answer]
Good point, thanks @kr4lt !

EqualStreetNames project has been added to <https://wiki.openstreetmap.org/wiki/Organised_Editing/Activities>
